test/BUILD.gn

```diff
@@ -98,6 +98,7 @@ test("brave_unit_tests") {
     "//brave/browser/brave_stats/brave_stats_updater_unittest.cc",
     "//brave/browser/browsing_data/brave_browsing_data_remover_delegate_unittest.cc",
     "//brave/browser/download/brave_download_item_model_unittest.cc",
+    "//brave/browser/misc_metrics/doh_metrics_unittest.cc",
```

Comment:
i probably missed this during previous reviews, but we shouldn't populate this section, rather tests should go to their own  BUILD.gn in corresponding directories. Can you please do a follow-up for metrics and time_period storage?

Reply:
fixed, will make follow-up PRs for other components